### DISCUSSION TITLE
github撰写内容时的一些快捷键

### DISCUSSION THREAD

**Discussion by jsishsjsj** [share121/inter-knot#2788] [其他]
![29093112nr7z](https://github.com/user-attachments/assets/14633595-22c1-4fd5-94ba-12591b933742)

![Screenshot_2024_0818_225452](https://github.com/user-attachments/assets/24585189-e055-4b85-9fb4-f4a31eb4482e)

这是功能的汉化（仅供参考）：

1. **Quote** - 引用
2. **Code** - 代码
3. **Link** - 链接
4. **Numbered list** - 编号列表
5. **Unordered list** - 无序列表
6. **Task list** - 任务列表
7. **Attach files** - 附加文件
8. **Mention** - 提及
9. **Reference** - 引用
10. **Slash commands** - 斜杠命令

以下是每个菜单选项在 Markdown 编辑器中的效果和使用说明：

1. **Quote (引用)**: 
   - 效果：将文本格式化为引用样式。
   - 使用方法：选中要引用的文本后点击“Quote”选项，或者在行首手动添加 `>` 符号。
   - 示例：
     ```markdown
     > 这是一个引用。
     ```

2. **Code (代码)**:
   - 效果：将文本格式化为代码片段或代码块。
   - 使用方法：选中代码后点击“Code”选项，或者在单行代码前后添加反引号 `` `code` ``，多行代码用三个反引号包围。
   - 示例：
     ```markdown
     `这是行内代码。`

     ```
     这是一个代码块。
     ```
     ```

3. **Link (链接)**:
   - 效果：将文本转换为超链接。
   - 使用方法：点击“Link”选项，然后填写显示文本和URL，或者手动使用 `[显示文本](URL)` 语法。
   - 示例：
     ```markdown
     [访问Google](https://www.google.com)
     ```

4. **Numbered list (编号列表)**:
   - 效果：创建一个有序列表。
   - 使用方法：点击“Numbered list”后，自动生成编号列表，或者在每行前加上 `1.` `2.` 依次类推。
   - 示例：
     ```markdown
     1. 第一项
     2. 第二项
     3. 第三项
     ```

5. **Unordered list (无序列表)**:
   - 效果：创建一个无序列表。
   - 使用方法：点击“Unordered list”后，自动生成无序列表项，或者在每行前加上 `-` 或 `*`。
   - 示例：
     ```markdown
     - 第一项
     - 第二项
     - 第三项
     ```

6. **Task list (任务列表)**:
   - 效果：创建一个任务列表，可以包含可勾选的复选框。
   - 使用方法：点击“Task list”后生成任务列表项，或者手动输入 `- [ ]` 表示未完成任务，`- [x]` 表示已完成任务。
   - 示例：
     ```markdown
     - [ ] 待办事项1
     - [x] 已完成事项2
     ```

7. **Attach files (附加文件)**:
   - 效果：允许你附加文件并将其链接到 Markdown 文档中。
   - 使用方法：点击“Attach files”选项，选择文件并上传，生成文件的下载链接。
   - 示例：
     ```markdown
     [下载文件](链接到文件)
     ```

8. **Mention (提及)**:
   - 效果：在协作环境中提及某个用户，通常用于评论或任务分配。
   - 使用方法：点击“Mention”后选择或输入要提及的用户名，生成 `@username` 形式的提及。
   - 示例：
     ```markdown
     @user 请查看此问题。
     ```

9. **Reference (引用)**:
   - 效果：用于引用文档或内容的其他部分。
   - 使用方法：点击“Reference”后选择要引用的内容或部分，生成一个指向该部分的链接。
   - 示例：
     ```markdown
     [参考部分](#部分标题)
     ```

10. **Slash commands (斜杠命令)**:
    - 效果：使用斜杠命令可以快速插入特定功能或格式，如 `/table` 生成表格，`/image` 插入图片等。
    - 使用方法：点击“Slash commands”或者在文档中直接输入 `/` 然后选择或输入命令。
    - 示例：
      ```markdown
      /image
      ```
图片中 **H** 按钮右边的两个按钮通常代表 **B** 和 **I**，分别用于加粗（Bold）和斜体（Italic）格式化文本。

### **B（加粗）** 按钮：
- **功能**: 将选中的文本加粗显示。
- **使用方法**: 选中要加粗的文本后点击 **B** 按钮，或者手动使用 Markdown 语法 `**加粗文本**`。
- **效果**:
  ```markdown
  **这是加粗的文本**
  ```
  显示为：**这是加粗的文本**

### **I（斜体）** 按钮：
- **功能**: 将选中的文本以斜体形式显示。
- **使用方法**: 选中要斜体的文本后点击 **I** 按钮，或者手动使用 Markdown 语法 `*斜体文本*` 或 `_斜体文本_`。
- **效果**:
  ```markdown
  *这是斜体的文本*
  ```
  显示为：*这是斜体的文本*

### 总结：
- **B** 按钮用于将文本加粗，使其更为突出。
- **I** 按钮用于将文本斜体化，通常用于强调特定内容。

图片中的 **H** 按钮通常代表 “Header”（标题）的缩写。在 Markdown 编辑器中，**H** 按钮用于插入或更改标题（即不同级别的标题）。

### 功能和使用方法：
- 点击 **H** 按钮后，通常会弹出一个下拉菜单，让你选择标题的级别，例如 `H1`、`H2`、`H3` 等。
- 在 Markdown 语法中，标题级别由 `#` 的数量来决定：
  - `#` 表示一级标题（H1）
  - `##` 表示二级标题（H2）
  - `###` 表示三级标题（H3）
  - 依此类推，可以创建最高 `H6` 级别的标题。

### 示例：
- 一级标题 (H1):
  ```markdown
  # 这是一级标题
  ```
- 二级标题 (H2):
  ```markdown
  ## 这是二级标题
  ```
- 三级标题 (H3):
  ```markdown
  ### 这是三级标题
  ```

### 结果：
不同级别的标题通常在文档中会显示为不同大小和粗细的字体，用于结构化文档内容。

因此，**H** 按钮的作用是帮助你快速插入或调整文档中的标题。

如果想更加了解，请转到[luogu的markdown指南](https://help.luogu.com.cn/rules/academic/handbook/markdown)

### 小贴士

1. 绳网发帖指南：#999 
2. [github中文化插件](https://github.com/maboloshi/github-chinese?tab=readme-ov-file)

**Comment by isle200inf** [comment]
建议在例子后面放上渲染效果<img src="https://bbs-static.miyoushe.com/static/2024/07/31/fdc5e0e30e577fb208f29effa93510ed_8442612635207222552.png" alt="云璃 期待" width="75" >

**Reply by isle200inf** [reply]
而且第八点全都放到代码块里了

**Comment by zhizhizhiwang** [comment]
有需要可以去看luogu的markdown
[指南](https://help.luogu.com.cn/rules/academic/handbook/markdown)

**Reply by zhizhizhiwang** [reply]
稍微了解一下还是够的（

**Reply by isle200inf** [reply]
<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >

**Comment by Maple013** [comment]
善/
